lib/taiko.js

```diff
@@ -1255,16 +1255,23 @@ async function mouseAction(selector, action, coordinates, options = {}) {
  *
  * @param {selector|string} selector - A selector of an element to scroll to.
  * @param {Object} options
+ * @param {boolean} [options.waitForNavigation=true] - Wait for navigation after the goto. Default navigationTimeout is 30 seconds to override pass `{ navigationTimeout: 10000 }` in `options` parameter.
  * @param {string[]} [options.waitForEvents = []] - Events available to wait for ['DOMContentLoaded', 'loadEventFired', 'networkAlmostIdle', 'networkIdle', 'firstPaint', 'firstContentfulPaint', 'firstMeaningfulPaint', 'targetNavigated']
- *
+ * @param {number} [options.navigationTimeout=30000] - Navigation timeout value in milliseconds for navigation after click.
+ * @param {number} [options.waitForStart = 100] - time to wait for navigation to start. Accepts value in milliseconds.
+ * @param {Object} options.alignments
+ * @param {string} [options.alignments.block = 'nearest'] - Defines vertical alignmen.
+ * @param {string} [options.alignments.inline = 'nearest'] - Defines horizontal alignmen.
+ * 
  * @returns {Promise<void>}
  */
 module.exports.scrollTo = async (selector, options = {}) => {
```

Comment:
It will be good to add an example on how to pass the options and link to
https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView for reference.

Reply:
Thank you for your review, I tied resolve and accepting your mentions.